a/asttokens/asttokens_ubi_8.4.sh

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : asttokens
+# Version       : v2.0.5
+# Source repo   : https://github.com/gristlabs/asttokens
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Gururaj R Katti <Gururaj.Katti@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+export ASTTOKENS_SLOW_TESTS=1
+
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "asttokens" ] ; then
+  rm -rf asttokens
+fi
+
+# Dependency installation
+sudo dnf install -y python39 git wget gcc python39-devel
+
+# Download the repos
+git clone https://github.com/gristlabs/asttokens
+
+# Build and Test asttokens
+cd asttokens
+git checkout $VERSION
+
+ret=$?
+
+if [ $ret -eq 0 ] ; then
+ echo "$Version found to checkout "
+else
+ echo "$Version not found "
+ exit
+fi
+
+pip3 install --upgrade setuptools>=44 wheel setuptools_scm[toml]>=3.4.3 pep517
+pip3 install --upgrade coveralls
+
+ret=$?
+
+if [ $ret -ne 0 ] ; then
```

Comment:
We can use `set -eu` at the start of the script, instead of having return value checks multiple times.

Reply:
Hi @amitsadaphule, thanks for the comments. The issue in putting "set -eu" at the start is that, it makes the script fail for if conditions such as "if [ -z "$1" ]; when we don't pass argument. Do you want me to put "set -eu" after this "if condition" and after the VERSION checkout ?